warmup.py

```diff
@@ -240,17 +251,39 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        for i, point in enumerate(points[:-1]):
+            neighbors = np.array(points[i+1:])
+            for neighbor in neighbors:
+                dist_vect = np.absolute(point - neighbor)
```

Comment:
No need to take an absolute value, the wrap operation works even with negative values in the vector.